### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Scriptkiddi**
Working nginx suckaaaaas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+## Example Playbook
+
+Configure a served_domain like 
+
+
+### Playbook
+
+```yml
```
Comment (SlothOfAnarchy) [inline README.md]: remove empty chapter

File: README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+
+### Vars
+
+```yml
+domain_suffixe:
+  - stuvus.uni-stuttgart.de.
+  - stuvus.de.
+
+domain_preffixe:
```
Comment (SlothOfAnarchy) [inline README.md]: typo in `preffixe`
Also, you use `prefixes` below

File: meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+galaxy_info:
+  author: Fritz Otlinghaus
+  description: nginx ansible role
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: CC-BY-SA
+
+  min_ansible_version: 2.2
+
+  github_branch: master
```
Comment (SlothOfAnarchy) [inline meta/main.yml]: remove

File: tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+- name: Update repositories cache and install "nginx" package
```
Comment (SlothOfAnarchy) [inline tasks/main.yml:2]: update cache missing

File: templates/nginx.conf.j2 [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        {{- domain -}}
+    {%- endif -%}
+{%- endmacro -%}
+{%- macro build_domain_list(domain, separator) -%}
+    {%- if domain[-1] == '.' -%}{#- domain is already a full qualified domain name -#}
+        {#- output domain name with appended prefix and suffix, with given separator and without leading dot -#}
+        {{- separator -}}{{ domain[:-1] }}
+    {%- else -%}
+        {%- for prefix in domain_prefixes | default(['']) -%}{#- append all domain prefixes -#}
+        {%- for suffix in domain_suffixes | default(['']) -%}{#- append all domain suffixes -#}
```
Comment (SlothOfAnarchy) [inline templates/nginx.conf.j2]: please indent

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Scriptkiddi) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Scriptkiddi) [reply README.md]: done

File: meta/main.yml (hunk 3 above)
Comment (Scriptkiddi) [reply meta/main.yml]: done

File: README.md [hunk 6]
```diff
@@ -57,21 +57,14 @@ See the [nginx doc](https://nginx.org/en/docs/http/ngx_http_core_module.html) fo
...
 ### Vars
 
 ```yml
-domain_suffixe:
+domain_suffixes:
   - stuvus.uni-stuttgart.de.
   - stuvus.de.
 
-domain_preffixe:
+domain_prefixes:
```
Comment (SlothOfAnarchy) [inline README.md]: are you using the changed vars correctly?

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**